Makefile

```diff
@@ -41,7 +42,8 @@ coverage: ## Run pytest with coverage report.
 .PHONY: preparedocs
 preparedocs: ## move file from the sphinx-starter-pack to docs folder
 	cp docs/sphinx-starter-pack/.sphinx/_static/* docs/_static
-	cp -R docs/sphinx-starter-pack/.sphinx/_templates docs/_templates
+	mkdir docs/_templates || true
```

Comment:
is the `|| true` for the case where `docs/_templates` already exists? If so, you can use `mkdir -p docs/_templates` instead, which has the advantage of not failing if the dir already exists but failing for any other problem, like 'permission denied'

Reply:
tbh, I don't remember why I did that xD

but your suggestion is correct. I'll change it